### PR TITLE
Made daily default on stats page

### PR DESCRIPTION
**Pull request by Henrymarks1**
- Changed the default Aggregation emun to be the value of the daily cases
- Changed the order of the daily and total values so it loops through the daily first, intern switching the default stat.

## Screenshots
Simulator Screen Shot - iPhone 12 Pro Max - 2020-10-26 at 12 56 31

<details>
<summary>Screenshots</summary>
Add any relevant before/after screenshots here
</details>

## How did you test the change?

- [x] iOS Simulator
- [x] iOS Device
- [ ] Android Simulator
- [ ] Android Device
- [ ] `curl` to a dev App Engine server
- [ ] other, please describe

## Checklist:

- [ ] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

**Comment by Henrymarks1** [comment]
Was unable to figure out how to change to the daily count on the main page

File: client/lib/pages/main_pages/recent_numbers.dart [hunk 1]
```diff
@@ -179,9 +179,9 @@ class _RecentNumbersPageState extends State<RecentNumbersPage> {
   Map<DataAggregation, Widget> _buildSegmentControlChildren(
       BuildContext context, DataAggregation selectedValue) {
     var valueToDisplayText = {
+      DataAggregation.daily: 'Daily',
       DataAggregation.total: S.of(context).latestNumbersPageTotalToggle,
       // TODO: This string changed, so we must change regenerate translation keys someday.
```
Comment (theswerd) [inline client/lib/pages/main_pages/recent_numbers.dart:184]: Remove this line, and add it up above the text **Daily** on line 182
```suggestion
```

**Review by theswerd — CHANGES_REQUESTED** [review]
See notes -- mostly LGTM

File: client/lib/pages/main_pages/recent_numbers.dart (hunk 1 above)
Comment (Henrymarks1) [reply client/lib/pages/main_pages/recent_numbers.dart]: @theswerd Made the recommended change!

**Comment by brunobowden** [comment]
Thanks Henry. I’ll have a look within the hour or so

File: client/lib/pages/main_pages/recent_numbers.dart (hunk 1 above)
Comment (theswerd) [reply client/lib/pages/main_pages/recent_numbers.dart]: Halfway done - you need to add it back above the unlocalized text

**Review by brunobowden — APPROVED** [review]
@Henrymarks1 thanks

File: client/lib/pages/main_pages/recent_numbers.dart [hunk 2]
```diff
@@ -179,9 +179,8 @@ class _RecentNumbersPageState extends State<RecentNumbersPage> {
   Map<DataAggregation, Widget> _buildSegmentControlChildren(
       BuildContext context, DataAggregation selectedValue) {
     var valueToDisplayText = {
-      DataAggregation.total: S.of(context).latestNumbersPageTotalToggle,
-      // TODO: This string changed, so we must change regenerate translation keys someday.
       DataAggregation.daily: 'Daily',
```
Comment (theswerd) [inline client/lib/pages/main_pages/recent_numbers.dart:182]:
```suggestion
      // TODO: This string changed, so we must change regenerate translation keys someday.
      DataAggregation.daily: 'Daily',
```

Comment (brunobowden) [reply client/lib/pages/main_pages/recent_numbers.dart:182]: How will people find the TODO? It's essential to use a similar naming for the TODO as used elsewhere. All the translation TODOs I've seen have started `// TODO: Localize`. Start with that and then append something.
```suggestion
      // TODO: Localize - need to regenerate translation keys as this string was changed
      DataAggregation.daily: 'Daily',
```

Comment (theswerd) [reply client/lib/pages/main_pages/recent_numbers.dart:182]: LGTM -- I just copied the old translation

Comment (Henrymarks1) [reply client/lib/pages/main_pages/recent_numbers.dart:183]: @theswerd I just put the old todo back

**Review by theswerd — APPROVED** [review]

**Comment by brunobowden** [comment]
@Henrymarks1 - thanks for the help. Please can you also comment “I Agree” here:
https://github.com/WorldHealthOrganization/app/issues/1645

**Pull request merged**